src/filters.rs

```diff
@@ -157,7 +398,7 @@ impl DocFilter {
     pub fn new(filter_config: Option<&FilterConfig>) -> Result<DocFilter, io::Error> {
         match filter_config {
             Some(filter_config) => match filter_config.syntax.as_deref() {
-                Some("jaq") => Ok(DocFilter::JqDocFilter(JqDocFilter::new(filter_config)?)),
+                Some("jq") => Ok(DocFilter::JqDocFilter(JqDocFilter::new(filter_config)?)),
```

Comment:
This fixes a mismatch in previous PR: I was using "jq" for syntax in python, but "jaq" here. Opting for "jq" since that's more well known.